### PR TITLE
Add Quantified Self

### PR DESCRIPTION
**Pull request by willempienaar**
**https://github.com/willempienaar/awesome-quantified-self**

**Awesome Quantified Self: Websites, resources, devices, wearables, applications, and platforms for self tracking.**
# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!
- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. _(For example: `Add Node.js`)_
- The list I added
  - [x] **has been around for at least 20 days,**
  - [x] is a non-generated Markdown file in a GitHub repo,
  - [x] is not a duplicate,
  - [x] only has awesome items with descriptions _(Awesome lists are curations of the best, not of everything)_,
  - [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
  - [x] has a Table of Contents section named `Contents` as the first section,
  - [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) _(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)_,
  - [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
  - [x] has been checked for inconsistent formatting, spelling, and grammar _(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)_,
  - [x] has been added at the bottom of the appropriate category.

Adding a link to the awesome quantified self repository: https://github.com/willempienaar/awesome-quantified-self

### CODE REVIEW

**Comment by davisonio** [comment]
Would be good if the repo had a description to replace 'No description or website provided.' e.g  'Websites, resources, devices, wearables, applications, and platforms for self tracking.'

May be nice to add a longer description at the top of the readme under the heading about what Quantified Self is. You could take the first few lines from https://en.wikipedia.org/wiki/Quantified_Self

Instead of using sub-bullets in the 'Applications and Platforms' section how about using a smaller heading (`### Aggregators & Dashboards` etc.)

**Comment by willempienaar** [comment]
Hi @davisonio 

Appreciate the feedback. I have added the descriptions and changed the bullets. Good suggestions by the way.

**Comment by sindresorhus** [comment]
I would like to see more articles / blog posts on the subject. Right now it's mostly just a list of apps and devices. Maybe also some talks on the subject (YouTube, TED), etc.

The descriptions should not be titleized, only the link titles. Makes it very hard to read them.

For the app descriptions, it would be useful to also include personal experiences about using them. How did it help that person. What is great about it.

`Quantified Self - Quantified Self Official Site.` => `Official Website`

The `Quantified Self` word in the intro description at the top should be linkified to https://en.m.wikipedia.org/wiki/Quantified_Self

File: readme.md [hunk 1]
```diff
@@ -450,6 +450,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Userscripts](https://github.com/brunocvcunha/awesome-userscripts) - Enhance your browsing experience.
 - [Pokémon](https://github.com/tobiasbueschel/awesome-pokemon) - Pokémon and Pokémon GO resources and tools.
 - [ChatOps](https://github.com/exAspArk/awesome-chatops) - Managing technical and business operations through a chat.
+- [Quantified Self](https://github.com/willempienaar/awesome-quantified-self)
```
Comment (sindresorhus) [inline readme.md]: Add a succinct description.

**Comment by sindresorhus** [comment]
ping

**Comment by willempienaar** [comment]
@sindresorhus I've added some blogs, articles, and talks. Also applied the other updates you suggested. In terms of personal experiences it is a bit tricky. I haven't personally used most of the devices/apps. We have an open issue to get some more input from actual users (https://github.com/willempienaar/awesome-quantified-self/issues/3).

**Comment by sindresorhus** [comment]
> In terms of personal experiences it is a bit tricky. I haven't personally used most of the devices/apps.

I'm curious how they're in the list if you haven't tried them. How do you know they're awesome?

**Comment by willempienaar** [comment]
> I'm curious how they're in the list if you haven't tried them. How do you know they're awesome?
> 
> The point of awesome lists are for people knowledgeable on a subject to share their experience with it.

I agree with you, but that is more realistic in a niche like a programming language or framework. The QS space is very varied, so that criteria would limit the list only to professional reviewers when it comes to devices.

Currently the way I curate the list is by monitoring sites like the ([QS Forum](https://forum.quantifiedself.com/), [AlternativeTo](http://alternativeto.net/), Reddit, and Quora. Based on suggestions, usage, and recommendations I try to gauge the strength of a technology or application.

The goal is to get some collaboration on the list (and descriptions) if there is enough merit to it.

**Comment by sindresorhus** [comment]
> Currently the way I curate the list is by monitoring sites like the QS Forum, AlternativeTo, Reddit, and Quora. Based on suggestions, usage, and recommendations I try to gauge the strength of a technology or application.

Makes sense, but then you could at least use some of the review comments from there.

File: readme.md [hunk 2]
```diff
@@ -455,9 +455,10 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
...
 - [Falsehood](https://github.com/kdeldycke/awesome-falsehood) - Falsehoods programmers believe in.
 - [Domain-Driven Design](https://github.com/heynickc/awesome-ddd) - Software development approach for complex needs by connecting the implementation to an evolving model.
-
+- [Quantified Self](https://github.com/willempienaar/awesome-quantified-self) - Quantified Self resources, apps, devices, and more.
```
Comment (sindresorhus) [inline readme.md]: It should be a succinct description of what quantified self is, not what the list contains.

You also have duplicated the entry.

File: readme.md [hunk 3]
```diff
@@ -457,7 +457,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [ChatOps](https://github.com/exAspArk/awesome-chatops) - Managing technical and business operations through a chat.
 - [Falsehood](https://github.com/kdeldycke/awesome-falsehood) - Falsehoods programmers believe in.
 - [Domain-Driven Design](https://github.com/heynickc/awesome-ddd) - Software development approach for complex needs by connecting the implementation to an evolving model.
-
```
Comment (sindresorhus) [inline readme.md]: Don't remove the newline.

**Pull request merged**